src/App.vue

```diff
@@ -298,7 +294,6 @@ export default {
     if (!this.isAuthenticated) return
 
     // ...otherwise proceed
-    await this.startTokenService()
```

Comment:
The old code started the token service only after the user was logged in, but that's a catch-22 because it syncs the session, which can log in (or log out) the user.

Also, the session needs to be synced (across domains) before any routing happens (since the router may redirect to a signin page).

So, this is moved to main.ts.

Reply:
Does this apply to our other UI's as well?
My hunch is that we similar elsewhere as well.

Reply:
Yes, this will apply to our other UIs where we init the token service in App instead of main.

Reply:
Actually, Namerequest UI is already done this way. (That's what I copied from.)